pavelib/assets.py

```diff
@@ -447,6 +449,10 @@ def update_assets(args):
         '--watch', action='store_true', default=False,
         help="Watch files for changes",
     )
+    parser.add_argument(
```

Comment:
Ok, I'm hearing you. On my end I'd like to pipe them to a file so Jenkins output is still readable, but we have the logs anyway. Would it be too much to have our cake and eat it as well? 

Say, if someone puts debug on, it'll pipe to console, yet we can still specify a log file? Too confusing that way?

I'll continue thinking on it but would appreciate your reaction/thoughts.